product_docs/docs/biganimal/release/overview/updates.mdx

```diff
@@ -8,6 +8,10 @@ EDB reserves the right to upgrade customers to the latest minor version without
 
 In some cases these updates might terminate existing network connections to your clusters. If that happens, the outage is typically less than 30 seconds. Be sure your applications are configured to automatically reconnect when connections are interrupted. Most modern database libraries do this by default. 
 
+## Maintenance window changes 
+
+As a Project Owner or Project Editor, you can ensure the stability and security of your databases by setting or editing maintenance windows for your clusters. To do this, enable the Custom Maintenance Window option and use the dropdown menus to select a day and time that works best for you. If you don't set a maintenance window, BigAnimal will assign a random one for you, which may not be ideal for your needs. Keep in mind that the 60-minute maintenance window specifies the start time of maintenance events. Additionally, BigAnimal will provide a notification message on the Cluster List four hours prior to the scheduled maintenance time, reminding you to prepare for the upcoming maintenance window by saving your work and closing any open connections.
```

Comment:
I would rewrite this to get rid of some of the extra info that is already included in Creating a cluster and instead just link to it. So something like:

"You can ensure the stability and security of your databases by setting maintenance windows for your clusters. For more information, see Maintenance(link to Creating a cluster section)."

The rest is included in Creating a cluster so it can be deleted.